File: Conceptos Basicos/ejercicio06.py
#6.Implementa una función que encuentre el máximo y mínimo de un arreglo 
# sin usar funciones predefinidas.
import random
array = []

for i in range(0,5):
    array.append(random.randint(0,100))

low = array[0]
high = array[0]
for i in range(0,len(array)):
    if(array[i]<low):
        low=array[i]
    
    if(high<array[i]):
        high=array[i]
    

print(array)
print(f"El numero de menor valor es {low} y el de mayor {high}")


#Opcion 2 

maximo =max(array)
menor = min(array)
print(f"El numero de menor valor es {menor} y el de mayor {maximo}")

File: Conceptos Basicos/ejercicio06.java

//6.Implementa una función que encuentre el máximo y mínimo de un arreglo 
//sin usar funciones predefinidas.
import java.util.Random;

public class ejercicio06 {
    public static void main (String[] arg){
        
        int array []= new int[5];
        Random rand = new Random();
        
        for (int i=0;i<array.length;i++){
            array[i]= rand.nextInt(100);
            System.out.println(array[i]);
        }
        
        int low=array[0];
        int high=array[0];

        for (int i=0;i<array.length;i++){
            if(array[i]<low){
                low=array[i];
            }

            if(array[i]>high){
                high=array[i];
            }
        }

        System.out.println("El menor numero es:"+low+" El mayor numero:"+high);

    }
}
